docker.sh

```diff
@@ -6,10 +6,6 @@ cd $(dirname $0)
 IMAGE=ownmailbox-image
 CONTAINER=ownmailbox
 
-docker() {
-    sudo docker "$@"
-}
-
```

Comment:
On the page you linked they suggest to create a docker group that is allowed to execute docker as sudo.

> To avoid having to use sudo when you use the docker command, create a Unix group called docker and add users to it.

Further down it then explains that calling `docker` doesn't need an explicit `sudo`.
This is how it should be done so that the `./docker.sh` script doesn't need it.

The problem here is, that forcing a `sudo` in the script can't be undone, so for me as an OSX user there is no way of executing it as non-sudo. Without the group, you'd just call `sudo ./docker.sh` if you needed it, the same way that you would need to call `sudo docker` when executing any docker command.

Makes sense, no?